### ISSUE TITLE
Proxy support

### ISSUE THREAD

**Issue by AshtarSheran** [evernote/evernote-sdk-python#2]
Is it possible to run test script behind http/https proxy?

**Comment by npassaro** [comment]
Yeah major setback for the likes of us that work behind corporate proxies in linux :cry:

**Comment by mwilck** [comment]
Eventually this is a python_thrift issue. I encourage everyone encountering the problem to vote for https://issues.apache.org/jira/browse/THRIFT-3798 and/or https://issues.apache.org/jira/browse/THRIFT-3507. Once it's fixed upstream, Evernote could pull in the updated thrift libraries more easily.
